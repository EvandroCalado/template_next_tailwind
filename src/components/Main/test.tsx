import { render, screen } from '@testing-library/react';
import { Main } from '.';

describe('Main', () => {
  it('should render correctly', () => {
    render(<Main />);

    const main = screen.getByRole('heading', { name: /Main/i });
    expect(main).toBeInTheDocument();
  });

  it('should render a snapshot', () => {
    const { container } = render(<Main />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
