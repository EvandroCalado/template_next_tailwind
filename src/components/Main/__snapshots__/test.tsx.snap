// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Main > should render a snapshot 1`] = `
<div>
  <h1>
    Main
  </h1>
</div>
`;

exports[`Main > should render correctly 1`] = `
<div>
  <h1>
    Main
  </h1>
</div>
`;
